src/App/Pages/Send/SendsPage.xaml.cs

```diff
@@ -12,15 +13,22 @@ public partial class SendsPage : BaseContentPage
         private SendsPageViewModel _vm;
         private bool _hasFocused;
 
-        public SendsPage(Func<SendView, bool> filter, bool type = false)
+        public SendsPage(Func<SendView, bool> filter, SendType? type = null)
```

Comment:
Would suggest moving away from nullable types as much as possible on new developments so we can eventually turn on [C# 8.0 (non) nullable references](https://endjin.com/blog/2020/04/dotnet-csharp-8-nullable-references-non-nullable-is-the-new-default). For enums in general, we should add a `Null` case ideally as the first or `= 0` field as that will be the default value used if a serialisation goes wrong. 

E.g:

```
    public enum SendType
    {
        Null,
        Text,
        File,
    }
```

Reply:
Can we please wait and not add `null` enum types and stick with `Nullable<t>` for the time being, until C# 8.0 has null references in-place or we're using it? Easier to refactor as well.